### DISCUSSION TITLE
Lighthouse with multi-ranger

### DISCUSSION THREAD

**Discussion by Noel700** [bitcraze/discussions#706] [Q&A]
Hi, 

I'm very new on using the crazyflie ecosystem. I wanna start a project where I wanna use the Lighthouse deck in Combination with the multi-ranger deck. I've read a paper which states that these two decks can't be combined. I imagine that the multi-ranger will block the sensors of the Lighthouse if it's on top. Vice versa the Lighthouse on top will block the vertical sensor of the multi-ranger.
Is this true and is this the only limitation if i combine these two decks? If so i will just ignore the vertical readout from the multi-ranger.
Thanks for any thoughts or experience on this.

Regards
Noel

**Comment by knmcguire** [comment]
So according to this table it should work: https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck---deck. There should not be any pin incompatibility at least

But this is not something we have tried often though. Do you have a lighthouse system and a multiranger so that you can try yourself already? If not we can give it a quick check.

**Reply by Noel700** [reply]
Thanks for your reply. 
Wasn't aware of this table but it will help me a lot. I currently have a lighthouse system and a multiranger at my university and we will start working on it very soon. We are currently waiting for the crazyradio to be delivered. So we will start working with the crazyflie within the next few weeks :)
